models/dex/dex_trades.sql

```diff
@@ -3,40 +3,10 @@
         post_hook='{{ expose_spells(\'["ethereum"]\',
                                 "sector",
                                 "dex",
-                                \'[""jeff-dude""]\') }}'
+                                \'["jeff-dude", "bh2smith"]\') }}'
         )
 }}
-SELECT *
-FROM
-(
-        SELECT
-            blockchain,
-            project,
-            version,
-            block_date,
-            block_time,
-            token_bought_symbol,
-            token_sold_symbol,
-            token_pair,
-            token_bought_amount,
-            token_sold_amount,
-            token_bought_amount_raw,
-            token_sold_amount_raw,
-            amount_usd,
-            token_bought_address,
-            token_sold_address,
-            taker,
-            maker,
-            project_contract_address,
-            tx_hash,
-            tx_from,
-            tx_to,
-            trace_address,
-            evt_index,
-            unique_trade_id
-        FROM {{ ref('uniswap_trades') }}
-        /*
-        UNION
-        <add future protocols here>
-        */
-)
\ No newline at end of file
+
+SELECT "AMM" as category, * from {{ ref('dex_amm_trades') }}
+UNION
+SELECT "Aggregator" as category, * from {{ ref('dex_aggregator_trades') }}
```

Comment:
Here is the ❤️ of the proposed structure.

`dex_trades` becomes this table that include an additional column "category", we refer to what were previously "dex" in v1 as "amm" (aka individual liquidity pool) and "aggregator" for aggregator. 

To defend the rational for using AMM here, is that when one goes to a platform like 1inch, in the interface they might refer to this as a DEX (a decentralized platform/protocol facilitating exchange). Similarly for CoW Protocol. Both of these fall under the category of aggregator, but it would not be incorrect to refer to them as DEXes. However, it would be incorrect to refer to them as AMMs.

Reply:
i may have misunderstood the original conversation around this, but i thought the goal was to split "amm" and "aggregator" -- which we are -- but then not to end up union'ing back into one table, since the confusion on v1 engine was having both in one table. i must have interpreted it as the two objects stay separate in v2 & you can join them together on certain fields to get a full picture.

i'm no expert in the dex sector, so happy to hear otherwise.

Reply:
This union has been removed.